### ISSUE TITLE
[Bug]: Paper catalog export - some geo areas not parsed correctly

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks#3392]
### Steps to reproduce the bug

```bash
1.When I export to paper catalog
2.Some geographic areas show up as "unparsed" and not in the correct geographic area section in the exported catalog.
3. Individual areas will be linked below as they are found. 
...
```

### Screenshot

![image](https://user-images.githubusercontent.com/11411328/235218608-f995f304-2073-41a0-a751-2c45b4ae5815.png)

### Expected behavior

All geo areas should be correctly parsed into the appropriate areas

### Environment

Production

### Version

0.32.2

### Browser Used

Chrome

**Comment by tmcelrath** [comment]
Fair enough - but they are provided, and meet our needs. If we shouldn't use them then that needs to be inferred somewhere. 

And/or, we need a way to batch change geographic areas.

**Comment by tmcelrath** [comment]
I should also note, not ALL of the TDWG names are unparsable, just a subset. Also, most of these are very easy to associate with a current geopolitical unit (e.g. Guatemala).

**Comment by mjy** [comment]
Points take @tmcelrath.  In fact there is a crude new task coming with the mapping updates that looks at the distribution of which gazetteer sources are used, and (very crudely) lets you find alternatives.  So as you note we anticipate people wanting to switch, and should have more power to do so soon.

**Comment by Omias** [comment]
Reunion https://sfg.taxonworks.org/geographic_areas/20039

**Comment by mjy** [comment]
There are now ways to batch update the GeographicArea of both CollectingEvent and AssertedDistribution.

**Comment by tmcelrath** [comment]
I think there are still GeoAreas without alternatives that do not parse properly.

**Comment by mjy** [comment]
I updated the general algorithim and there were massive improvments. I also regenerated all the data for INHS.  If there are issues at this point then it is with the underlying data.  

Geo-roadmap wise we are implementing Gazeteer (user creatable shapes) first, then will explore completely revamping our GA authorities.  The former hopefully before TWT (May 2025), the later open-ended.
